pkgs/development/python-modules/dulwich.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchurl
+, pkgs, gevent, geventhttpclient, mock, fastimport }:
```

Comment:
the reasoning behind passing individual packages is to make it easier to overwrite them?